spec.html

```diff
@@ -29005,17 +29028,26 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Return the result of evaluating |VariableStatement|.
         </emu-alg>
-        <emu-grammar>ExportDeclaration : `export` Declaration</emu-grammar>
+        <emu-grammar>ExportDeclaration : DecoratorList? `export` Declaration</emu-grammar>
         <emu-alg>
-          1. Return the result of evaluating |Declaration|.
+          1. If |DecoratorList?| is present, then
```

Comment:
```suggestion
          1. If |DecoratorList| is present, then
```
I personally am neutral on whether the `?` should be included in general, but right now the prevailing style is that it is not. (See for example [ForLoopEvaluation](https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-declarations.html#sec-runtime-semantics-forloopevaluation).)

Reply:
I can change that, but there are other cases of `|DecoratorList?|` in the main PR we'll need to address at some point as well.

Reply:
Feel free to leave this as-is and address it as part of the main PR if you'd prefer, then.